build/SignToolData.json

```diff
@@ -0,0 +1,24 @@
+{
+    "sign": [
+        {
+            "certificate": "Microsoft402",
+            "strongName": "MsSharedLib72",
+            "values": [
+                "bin/Microsoft.DotNet.Archive/netstandard1.3/Microsoft.DotNet.Archive.dll",
+                "bin/Microsoft.DotNet.Archive/netstandard1.3/cs/Microsoft.DotNet.Archive.resources.dll",
```

Comment:
>cs [](start = 61, length = 2)

You can write it as 
```bin/Microsoft.DotNet.Archive/netstandard1.3/*/Microsoft.DotNet.Archive.resources.dll```

to match all cultures

Reply:
ah, yes, forgot Jared added that.